skyportal/handlers/api/annotation_services.py

```diff
@@ -120,14 +123,6 @@ def post(self, obj_id):
             }
 
             origin = f"{catalog}-{row['ra']}-{row['dec']}"
-            if (
-                (annotation_data['w1mpro'] - annotation_data['w2mpro'] > 0.6)
-                and (annotation_data['w1mpro'] - annotation_data['w2mpro'] < 1.7)
-                and (annotation_data['w2mpro'] - annotation_data['w3mpro'] > 2.2)
-                and (annotation_data['w2mpro'] - annotation_data['w3mpro'] < 4.5)
-            ):
-                annotation_data['comment'] = 'AGN?'
-
```

Comment:
We should think about a way to add this (very useful) functionality in a way that is config-definable (in a separate PR, of course).

Reply:
maybe something that checks... all annotations from the external services and applies them if they apply? or it could be like config.annotations.agn flag default to false and turns this check on if true?

Reply:
I was thinking more of some generic way to flag objects based on annotation data. 
You'd have to define the name of the flag (e.g., AGN), the set of origins (if not specified, look at all of them) and the data keys you want to look at. 
This is somewhat complex because you'd have to have a whole language to parse which operators and logical combinations are accepted by these definitions. It may be simple enough if you allow only a few operations (e.g., "color" will always look for two keys and subtract them) and always require all the conditions to be true to pass, in which case you can have a relatively simple interface. If you want to do something even more generic, it is probably best to just pull the annotation data for each source, run custom code, and push back whatever comment/annotation you want (i.e., a service call).

Reply:
@guynir42 It kind of reminds me what we did for the period(?) stuff... a bit, but these are obviously more intense.